app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,35 @@
+
+@import "bootstrap-sprockets";
+@import "bootstrap";
+/* universal */
+body {
+padding-top: 60px;
+}
```

file này sai indent rồi em, thụt vào nếu là block con nha
```suggestion
body {
  padding-top: 60px;
}
```